src/test/Sentry.Unity.Editor.Tests/EditorModeTests.cs

```diff
@@ -55,5 +58,18 @@ public void OptionsDsnField_WrongFormat_CreatesError()
             Assert.AreEqual(1, validationErrors.Count);
             Assert.NotNull(validationErrors.SingleOrDefault(e => e.PropertyName.Contains(nameof(SentryTestWindow.Options.Dsn))));
         }
+
+        // This test method has a side effect of creating 'link.xml' if file doesn't exits.
+        [Test]
+        public void SentryTestWindow_OpenAndLinkXmlCopied_Successful()
+        {
+            LogAssert.ignoreFailingMessages = true; // mandatory
+
+            // Open & Close window to trigger 'link.xml' logic
+            SentryTestWindow.Open().Dispose();
+
+            var linkXmlPath = $"{Application.dataPath}/Resources/{UnitySentryOptions.ConfigRootFolder}/link.xml";
+            Assert.IsTrue(File.Exists(linkXmlPath));
+        }

```

Comment:
What about we make sure it wasn't there to begin with. Or should we delete it before the test runs?

```suggestion
        public void SentryTestWindow_OpenAndLinkXmlCopied_Successful()
        {
            var linkXmlPath = $"{Application.dataPath}/Resources/{UnitySentryOptions.ConfigRootFolder}/link.xml";
            Assert.IsFalse(File.Exists(linkXmlPath)); // sanity check
            LogAssert.ignoreFailingMessages = true; // mandatory

            // Open & Close window to trigger 'link.xml' logic
            SentryTestWindow.Open().Dispose();

            Assert.IsTrue(File.Exists(linkXmlPath));
        }

```